lib/services/directions_service.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'network.dart';
```

Comment:
We use absolute imports project-wide so this should be an absolute import (per https://fuchsia.dev/fuchsia-src/development/languages/dart/style#do_import_all_files_within_a_package_using_relative_paths)

Reply:
Not possible as it's a package.